<launch>
  <!-- run rino node -->
  <node name="rino" pkg="rino" type="rino" output="screen">
  
    <param name="dataset" type="str" value="mulran" />
    <param name="keypoint_extraction" type="str" value="yang2024" />
    <param name="use_voxelization" type="bool" value="false" />
    <param name="viz_extraction" type="bool" value="false" />
    <param name="viz_matching" type="bool" value="false" />
    
    <param name="use_motion_deskew" type="bool" value="true" />
    <param name="use_doppler_compensation" type="bool" value="true" />
    <param name="viz_undistortion" type="bool" value="false" />
  </node>

  <!-- run RViz -->
  <node name="rviz" pkg="rviz" type="rviz" args="-d /home/yangsc4063/SLAM/Radar_SLAM/RINO_release/src/rino/rviz/rino_rviz.rviz" />
</launch>